rar.go

```diff
@@ -37,6 +38,23 @@ func Unrar(source, destination string) error {
 		} else if err != nil {
 			return err
 		}
+		
+		pathComponents := strings.Split(header.Name, "/")
```

Comment:
I have been testing on windows exclusively so far, so I can confirm that the forward-slash "/" is used on windows. Haven't tested on other platforms yet.